geocity/apps/submissions/models.py

```diff
@@ -271,6 +271,7 @@ class Meta:
         permissions = [
             ("read_submission", _("Consulter les demandes")),
             ("amend_submission", _("Traiter les demandes")),
+            ("edit_submission_validations", _("Modifier les validations")),
```

Comment:
Be consistent with `edit_submission` (only use one of these terms `Modifier` (my preference) or `Editer`)